tests/Core/File/GetTokensAsStringTest.php

```diff
@@ -102,7 +102,7 @@ public function testLengthBeyondEndOfFile()
      * Test getting a token set as a string.
      *
      * @param string     $testMarker     The comment which prefaces the target token in the test file.
-     * @param int|string $startTokenType The type of token(s) to look for for the start of the string.
+     * @param int|string $startTokenType The type of token(s) to look for the start of the string.
```

Comment:
This change is incorrect. The original may look strange with the double `for`, but is correct. (same below)